website/docs/getting-started.mdx

```diff
@@ -84,6 +137,70 @@ For now choose to use either JSON or Yaml. Below are examples of each that inclu
             - '/project-words.txt'
         ```
     </TabItem>
+    <TabItem value="esm" label="cspell.config.mjs">
+        ```javascript
+        'use strict';
+
+        import { defineConfig } from "cspell";
+
+        export default defineConfig({
+        $schema:
```

Comment:
`$schema` is only useful for `.json` files. It helps editors to to type checking on the file.